### PR TITLE
odim 3110:Adding ut for Get Aggregation Source in svc-aggregation 

### PR DESCRIPTION
**Pull request by jeevan-kamkar**

### CODE REVIEW

**Review by irappa-pattar — DISMISSED** [review]

**Review by a-ajith — DISMISSED** [review]

File: svc-aggregation/rpc/aggregator_test.go [hunk 1]
```diff
@@ -710,3 +723,119 @@ func TestAggregator_AddAggreagationSource(t *testing.T) {
...
+	}
+}
+
+func TestAggregator_GetAggregationSource(t *testing.T) {
+	defer func() {
+		common.TruncateDB(common.OnDisk)
+		common.TruncateDB(common.InMemory)
+	}()
+	req := agmodel.AggregationSource{
+		HostName: "10.24.0.14",
```
Comment (shivacharanms) [inline svc-aggregation/rpc/aggregator_test.go]: Change IP to more generic.

File: svc-aggregation/system/get_resource_test.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+	"testing"
+)
+
+func TestGetAggregationSourceCollection(t *testing.T) {
+	defer func() {
+		common.TruncateDB(common.OnDisk)
+		common.TruncateDB(common.InMemory)
+	}()
+	req := agmodel.AggregationSource{
+		HostName: "10.24.0.14",
```
Comment (shivacharanms) [inline svc-aggregation/system/get_resource_test.go]: Change IP to more generic

File: svc-aggregation/system/get_resource_test.go [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+	}
+}
+
+func TestGetAggregationSource(t *testing.T) {
+	defer func() {
+		common.TruncateDB(common.OnDisk)
+		common.TruncateDB(common.InMemory)
+	}()
+	req := agmodel.AggregationSource{
+		HostName: "10.24.0.14",
```
Comment (shivacharanms) [inline svc-aggregation/system/get_resource_test.go]: Change IP to more generic

File: svc-aggregation/system/get_resource_test.go [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+			name: "Postive Case",
+			args: args{
+				reqURI: "/redfish/v1/AggregationService/AggregationSource/123455",
+			},
+			want: response.RPC{
+				StatusCode: http.StatusNotImplemented, //replace with resp1 after the implementation is completed
+			},
+		},
+		{
+			name: "Postive Case",
```
Comment (shivacharanms) [inline svc-aggregation/system/get_resource_test.go]: Duplicate test case

**Review by shivacharanms — CHANGES_REQUESTED** [review]

File: svc-aggregation/system/get_resource_test.go (hunk 4 above)
Comment (jeevan-kamkar) [reply svc-aggregation/system/get_resource_test.go]: removed

File: svc-aggregation/system/get_resource_test.go (hunk 3 above)
Comment (jeevan-kamkar) [reply svc-aggregation/system/get_resource_test.go]: updated

File: svc-aggregation/system/get_resource_test.go (hunk 2 above)
Comment (jeevan-kamkar) [reply svc-aggregation/system/get_resource_test.go]: updated

File: svc-aggregation/rpc/aggregator_test.go (hunk 1 above)
Comment (jeevan-kamkar) [reply svc-aggregation/rpc/aggregator_test.go]: updated

**Review by shivacharanms — APPROVED** [review]

**Comment by jeevan-kamkar** [comment]
merging the pr after showing demo @shivacharanms and @bharath-b-hpe

**Review by bharath-b-hpe — APPROVED** [review]
LGTM

**Pull request merged**